### PR TITLE
Fixed displaying issue of footer component

### PR DESCRIPTION
**Pull request by mkajic20**
Fixed footer component so it displays properly on the page.

### CODE REVIEW

**Review by lgaljer20 — APPROVED** [review]
Looks good.

File: Software/smart-charger/src/components/Footer/Footer.jsx
Comment (irosic20) [inline Software/smart-charger/src/components/Footer/Footer.jsx:1]: You should remove unnecessary comments!

**Review by irosic20 — CHANGES_REQUESTED** [review]
Everything looks good, i just had 1 comment I'd like you to fix!

File: Software/smart-charger/src/components/Footer/Footer.jsx
Comment (mkajic20) [reply Software/smart-charger/src/components/Footer/Footer.jsx:1]: I've removed the comments, thank you!

**Review by irosic20 — APPROVED** [review]
Everything looks good now!

**Pull request merged**